autogen/agentchat/groupchat.py

```diff
@@ -464,7 +476,7 @@ def __init__(
             **kwargs,
         )
         # Store groupchat
-        self._groupchat = groupchat
+        self.groupchat = groupchat
```

Comment:
why this change?

Reply:
For nested agents, the only way to get access to the groupchat is through `_groupchat`, but I can't access it since it's a private variable. From my understanding, autogenstudio also needs access to the groupchat, so I thought it might be cleaner to change it to a public variable.

Reply:
The cleanest way would be to define a new property `groupchat`, you don't want this variable to be changed from the outside.